logging_config.js

```diff
@@ -1,15 +1,18 @@
 /* eslint-disable no-unused-vars, no-var */
 // Logging configuration
+// XXX When making any changes to this file make sure to also update it's React
+// version at ./react/features/base/logging/reducer.js !!!
```

Comment:
interesting, no way for these to be able to share a single file?

Reply:
there's no easy way and given how often this file is modified I thought a comment would be enough...

Reply:
My general motivation for base/logging/reducer.js includes the following:
* The reducer's default values are really necessary (at least when I put them there) because otherwise the logging is too verbose and hinders debugging.
* I didn't want to download logging_config.js on mobile because we were planning on merging the configs and we'd end up getting the logging_config at some point in the future anyway from (the merged) config.js.
* I couldn't `import` logging_config.js because it doesn't export anything.

Reply:
ok, personally i think fixing the causes of the last 2 bullets there would be preferable to duplicating the file and requiring all changes to be made in both places.

Reply:
I think everyone agrees it would be great to have all those issues fixed, but it will not happen as part of this PR. I was trying to look into the 3rd one and it's not easily fixable or at least I don't know how to fix it without spending few days on it. Given this file is updated once or few times a year it's not worth the effort at this point IMO.